src/utils/format.ts

```diff
@@ -0,0 +1,11 @@
+const dateFormatter = new Intl.DateTimeFormat('en-US', {
+  dateStyle: 'medium',
+});
+
+export function formatDate(date: Date) {
+  return dateFormatter.format(date);
+}
+
+export function formatDateRange(date1: Date, date2: Date) {
+  return formatDate(date1) + ' - ' + formatDate(date2);
+}
```

Comment:
you know what, I just realized that Intl.DateTimeFormat knows how to format ranges (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/formatRange).
But I'm not sure we want that in this context.

From my test, the best formatter would be:
```js
const dateRangeFormatter = new Intl.DateTimeFormat('en-US', {
  month: "short",
  day: "numeric"
});
// then
export function formatDateRange(date1: Date, date2: Date) {
  return dateRangFormatter.formatRange(date1, date2);
}
```

By not specifying the year, it would show up only when needed.

Can you please try that and see how this looks?

Reply:
Here is how it looks:

month: "short", day: "numeric"            |  dateStyle: 'medium'
:-------------------------:|:-------------------------:
![image](https://github.com/user-attachments/assets/4680deab-fc67-4ebd-8d85-b1383c861031)  |  ![image](https://github.com/user-attachments/assets/3c8e9b29-617a-46d1-a45e-f5d8eadef1a8)



I like how concise it is while still conveying the complete information.

Reply:
Yeah, I'd like the concise format better if this was showing up separate from the others.
But in this case, I find the format at the right easier to parse visually, because the same operation is present at the same location for every rows.
So let's keep `dateStyle: medium`, but thanks for trying it!